modules/http/src/main/scala/higherkindness/mu/http/protocol.scala

```diff
@@ -21,6 +21,13 @@ import org.http4s.HttpRoutes
 import org.http4s.server.blaze.BlazeServerBuilder
 import org.http4s.implicits._
 import org.http4s.server.Router
+import scala.annotation.StaticAnnotation
+
+object protocol {
```

Comment:
I was thinking about an standard package rather than an object

Reply:
Take a look at the name of the file, we already had `protocol.scala` (because this was cloned from the rpc part).